master/buildbot/reporters/gerrit_verify_status.py

```diff
@@ -226,6 +229,16 @@ def send(self, build):
                     reporter=reporter,
                     category=category,
                     duration=duration)
+
+                # The verify-status plugin will return 204 NO CONTENT
+                # on success.
+                if response.code != http_client.NO_CONTENT:
+                    try:
+                        body = yield response.text()
```

Comment:
`response.text()` returns unicode, while `response.content` returns bytes.
It's probably better to leave this as `response.text()` especially for Python 3,
otherwise we get into the bytes/unicode issues.